src/scsi/README.md

```diff
@@ -0,0 +1,43 @@
+# vhost-user-scsi
+
+This is a Rust implementation of a vhost-user-scsi daemon.
+
+## Usage
+
+Run the vhost-user-scsi daemon:
+
+```
+vhost-user-scsi -r /tmp/vhost-user-scsi.sock /path/to/image.raw /path/to/second-image.raw ...
+```
```

Comment:
With my testing it crapped out on me:

    ➜  ./target/debug/vhost-user-scsi -r /tmp/vhost-user-scsi.sock test.img
    [2021-12-07T11:46:48Z ERROR vhost_user_scsi] Error running daemon: HandleRequest(InvalidParam)
    [src/scsi/src/main.rs:391]

Reply:
Hmm, not easily reproducing this (AlmaLinux 8.5 host, self-built QEMU 6.2, Debian 11 "nocloud" qcow2 image as guest). Some thoughts:

* Setting `RUST_LOG=debug` will get you a better idea of when things go wrong.
* Does this happen as soon as you start `vhost-user-scsi`, or after QEMU connects?
* What QEMU version and guest OS are you running?